Content.Server/GameObjects/Components/Arcade/SpaceVillainArcadeComponent.cs

```diff
@@ -306,20 +334,23 @@ private bool CheckGameConditions()
             {
                 if ((_enemyHp <= 0 || _enemyMp <= 0) && (_playerHp > 0 && _playerMp > 0))
                 {
-                    UpdateUi("You won!", $"{_enemyName} dies.");
+                    _running = false;
+                    UpdateUi("You won!", $"{_enemyName} dies.", true);
```

Comment:
Those probably need loc too

Reply:
go no idea how to do this

Reply:
due to the parameters in the strings that is

Reply:
`Loc.GetString("{0} dies.", _enemyName);`

Reply:
will do